tests/models/test_regression.py

```diff
@@ -82,7 +82,7 @@ def __next__(self):
                         raise StopIteration()
             return MockIterator()
 
-        def do_nothing(self, data_path, batch_file_size, shuffle_data, mode):
+        def do_nothing(self, data_path, batch_file_size, shuffle_data, mode, pred_months):
```

Comment:
I don't want the dataloader to look for filenames (because there aren't any), and it does that during initialization. Instead, I overload the `__iter__` function to return an array, and overload the `__init__` function to do nothing